docs/adr/adr-007-data-retention.md

```diff
@@ -0,0 +1,196 @@
+# Decision Record: Data Retention
+
```

Comment:
Looks like this is missing Date from the template: https://github.com/alphagov/email-alert-api/blame/master/docs/adr/adr-005-record-architecture-decisions.md#L3